.gitignore

```diff
@@ -17,3 +17,7 @@ tags
 .bloop/
 metals.sbt
 .vscode
+
+# Macos
+.DS_Store
+.Trashes
```

Comment:
Since these are specific to your operating system (and not this project) they should go in your global `.gitignore` configuration